west.yml

```diff
@@ -51,7 +51,7 @@ manifest:
       revision: 900c7eea92e45adba1067e044ec705d4541c84a5
     - name: mcuboot
       repo-path: fw-nrfconnect-mcuboot
-      revision: 0cf7965a146bad1becb1e260481d0e32e853d38e
+      revision: pull/42/head
```

Comment:
@hakonfam : Doesn't this need to be sha-ified before the PR goes in?

Reply:
Yes, the push was a double push, but I'm not sure how we should do this. Since merging in mcuboot will break that before this is in and vice verca

Reply:
Maybe I should just revert the SHA and then merge this in first? That way we dont' brake anything. Then mcuboot can be merged at a later time

Reply:
What did we do last time? We merged with a PR reference here didn't we?

IIRC that worked out all right.